examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -146,8 +145,23 @@ def __process_worker_register(self, worker_id, input_value_json, response):
                 "Worker Id already exists in the database. \
                  Hence invalid parameter")
         return response
+
 # ------------------------------------------------------------------------------------------------
+    def validate_input_worker_status(self, input_json):
+        '''
+        Validates worker status in the input request
+        '''
+        try:
+            worker_status = input_json["params"]["status"]
+            status = int(worker_status)
+            if status < WorkerStatus.ACTIVE.value or status > WorkerStatus.COMPROMISED.value:
```

Comment:
Don't compare the range using value. What if we add one more status type to enum?
Use type conversion something like below.
try:
            worker_status = input_json["params"]["status"]
            status = int(worker_status)
            wo_status = WorkerStatus(status)
            return True
except:
            logger.error("Invalid parameter format or value found in the request")
            return False